Dockerfile

```diff
@@ -17,15 +17,12 @@ COPY . .
 RUN make VERIFY=$VERIFY all
 
 #############      base                                     #############
-FROM alpine:3.10.3 AS base
```

Comment:
Please, revert all of the changes in the Docker file as they breaks the built process.